docroot/themes/humsci/humsci_basic/src/scss/components/_local-footer.scss

```diff
@@ -17,6 +17,21 @@
 
   .menu-item {
     margin-bottom: hb-calculate-rems(14px);
+
+    @include grid-media-min('lg') {
+      font-size: hb-calculate-rems(18px);
+    }
+
+    @include hb-colorful {
+      a {
+        &:hover,
+        &:focus {
+          .fa-ext::before {
+            background: svg('<svg viewBox="0 0 12 12"><path d="M11.273 5.818v5.455H.727V.727h5.091m1.455 0h4v4m0-4L6.182 5.818" fill="transparent" stroke="#{hb-colorful-variation(primary)}" stroke-width="2" stroke-miterlimit="10"/></svg>') center center no-repeat;
```

Comment:
hmmm since this svg is used a lot, shall we move it to the settings.icons.scss?

Reply:
should we also be using the `hb-external-link-icon` mixin here instead? also on 78 below?

Reply:
`fa-ext` calls the `hb-external-link-icon` mixin when it is declared in settings/base.scss